tests/system/Config/DotEnvTest.php

```diff
@@ -61,7 +62,7 @@ public function testLoadsHex2Bin()
 		$dotenv = new DotEnv($this->fixturesFolder, 'encryption.env');
 		$dotenv->load();
 
-		$value = getenv('encryption.key');
+		$value = $_ENV['encryption.key'] ?? getenv('encryption.key');
```

Comment:
```suggestion
		$value = getenv('encryption.key');
```
and then `putenv('encryption.key')` before the line where we create new `DotEnv` object (if needed).